launchable/commands/record/tests.py

```diff
@@ -256,14 +279,31 @@ def send(payload: Dict[str, List]) -> None:
                     traceback.print_exc()
                     return
 
-            click.echo("Recorded {} tests".format(count))
             if count == 0:
                 if len(self.skipped_reports) != 0:
                     click.echo(click.style(
-                        "{} test reports were skipped because they were created before `launchable record build`.\nMake sure to run test after `launchable record build`.".format(len(self.skip_reports)), 'yellow'))
+                        "{} test reports were skipped because they were created before `launchable record build`.\nMake sure to run test after `launchable record build`.".format(len(self.skipped_reports)), 'yellow'))
```

Comment:
```suggestion
                        "{} test reports were skipped because they were created before `launchable record build`.\nMake sure to run tests after `launchable record build`.".format(len(self.skipped_reports)), 'yellow'))
```